src/modules/esl-toggleable/core/esl-toggleable.ts

```diff
@@ -167,12 +167,8 @@ export class ESLToggleable extends ESLBaseElement {
     if (this._trackHover === track) return;
     this._trackHover = track;
 
-    this.removeEventListener('mouseenter', this._onMouseEnter);
-    this.removeEventListener('mouseleave', this._onMouseLeave);
-    if (this._trackHover) {
-      this.addEventListener('mouseenter', this._onMouseEnter);
-      this.addEventListener('mouseleave', this._onMouseLeave);
-    }
+    this._trackHover ? this.$$on(this._onMouseEnter) : this.$$off(this._onMouseEnter);
+    this._trackHover ? this.$$on(this._onMouseLeave) : this.$$off(this._onMouseLeave);
```

Comment:
```suggestion
    track ? this.$$on(this._onMouseEnter) : this.$$off(this._onMouseEnter);
    track ? this.$$on(this._onMouseLeave) : this.$$off(this._onMouseLeave);
```
such declaration is more compact